standard/examples/instance_metadata_JSON.adoc

```diff
@@ -821,24 +818,22 @@ There are also links to the license information for the observation and forecast
             ],
             "links": [
                 {
-                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/position",
-                    "hreflang": "en",
-                    "rel": "data"
-                },
-                {
-                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/radius",
+                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T08:00:00Z",
                     "hreflang": "en",
-                    "rel": "data"
+                    "rel": "self",
+                    "type": "application/json"
                 },
                 {
-                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/area",
+                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T08:00:00Z?f=html",
```

Comment:
Just caught many instances of this (double protocol).  Maybe normalizing to https://example.org is better done in this PR?  Your call.

Reply:
@tomkralidis Thanks I had missed that